### DISCUSSION TITLE
asp-callback

### DISCUSSION THREAD

**Discussion by mkozen** [sinanbozkus/FormHelper#17] [Q&A] [answered]
asp-callback tag-helper'ini nasıl kullanabilirim? Amacım form submit edildikten sonra formdaki bir adet inputun değerini değiştirmek. Yardımcı olursanız çok sevinirim.

**Comment by sinanbozkus** [comment, marked as answer]
Merhaba,

Örneğin javascript tarafında FormCallback adında bir fonksiyonunuz var. Bunun form taginiz içerisinde asp-callback parametresine bu fonksiyonun adını vermeniz gerekiyor.

`<form asp-callback="FormCallback">`

daha sonra ilgili javascript methodu içerisine işleminizi gerçekleştirebilirsiniz. result içerisinde FormHelper'dan gelen değerler yer almaktadır. Dilerseniz ek değerlerde gönderebilirsiniz.

```
<script>
function FormCallback(result){
$("#inputAdi").val("yeniDeğer");
}
</script>
```

FormHelper kütüphanesini faydalı buluyorsanız lütfen star vererek destek olunuz 👍🏻
